react/src/components/ResetPasswordRequired.tsx

```diff
@@ -0,0 +1,177 @@
+import { Button, Form, Modal, Typography, Input, theme } from "antd";
+import { useWebComponentInfo } from "./DefaultProviders";
+import Flex from "./Flex";
+import { useTranslation } from "react-i18next";
+import { WarningTwoTone } from "@ant-design/icons";
+import { useMutation as useTanMutation } from "react-query";
+import { parse } from "path";
+import { useAnonymousBackendaiClient } from "../hooks";
+import { baiSignedRequestWithPromise } from "../helper";
+import { useEffect } from "react";
+// import * as ai from "../lib/backend.ai-cleint-esm_on_react";
+// import { useSuspendedBackendaiClient } from "./BackendaiClientProvider";
+
+// src/components/backend-ai-common-utils.ts
+export const passwordPattern = /^(?=.*\d)(?=.*[a-zA-Z])(?=.*[_\W]).{8,}$/;
+
```

Comment:
This is just a minor thought, but I wonder if we could store password patterns into a single component(just came to my mind), so that we could import and apply it and change it if the policy updates.

Reply:
Thank you for the suggestion. For now, I think this is okay to export the reg expression only, it's more flexibility in composition. But it would be better to move this pattern to another file.